Blockzilla/FullScreen.js

```diff
@@ -0,0 +1,40 @@
+var isFullScreenEnabled = document.fullscreenEnabled ||
```

Comment:
Don't forget to add the MPL license header. We need it on top of every source file.

Reply:
done